x-pack/plugins/security/server/authorization/disable_ui_capabilities.ts

```diff
@@ -19,7 +19,7 @@ export function disableUICapabilitiesFactory(
   authz: AuthorizationServiceSetup
 ) {
   const featureNavLinkIds = features
-    .map((feature) => feature.navLinkId)
+    .flatMap((feature) => [feature.navLinkId, ...feature.app])
```

Comment:
__optional nit:__ would you mind adding a comment here that explains why we use nav link IDs and app IDs? I took a look at the code before I read PR description I got confused a bit app IDs being a part of `featureNavLinkIds` :slightly_smiling_face: